Formula/netcdf-mpi.rb

```diff
@@ -0,0 +1,50 @@
+class NetcdfMpi < Formula
+  desc "The NetCDF C I/O library compiled with MPI parallel features enabled"
+  homepage "https://www.unidata.ucar.edu/software/netcdf/"
+  url "https://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-c-4.7.4.tar.gz"
+  sha256 "0e476f00aeed95af8771ff2727b7a15b2de353fb7bb3074a0d340b55c2bd4ea8"
+  license "BSD-3-Clause"
+
+  depends_on "hdf5-mpi"
+
+  conflicts_with "netcdf", :because => "netcdf-mpi is an MPI parallel build, use either netcdf or netcdf-mpi"
+
+  def install
+    # use the C compiler wrapper
+    ENV["CC"] = "mpicc"
+
+    # move files that NetCDF wants to place in libdir,
+    # because brew does not want them there. etc was
+    # recommended during the code review
+    inreplace "./Makefile.in",
+          "settingsdir = $(libdir)",
+          "settingsdir = #{etc}/netcdf-mpi"
+
+    system "./configure", "--enable-shared",
+                          "--enable-static",
+                          "--disable-dap",
+                          "--enable-netcdf-4",
+                          "--enable-parallel4",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+  end
+
+  test do
+    assert_equal "yes", shell_output("#{bin}/nc-config --has-parallel4").strip
+
+    (testpath/"test.c").write <<~EOS
+      #include <stdio.h>
+      #include "netcdf_meta.h"
+      int main()
+      {
+        printf(NC_VERSION);
```

Comment:
```suggestion
        puts(NC_VERSION);
```

Reply:
Because `NC_VERSION` is a macro, this test doesn't use any symbols from the `netcdf` library. Please call a function from the library.

Reply:
agree. I added 2 tests that use the install to write simple files in serial and parallel